src/misc/lv_style.c

```diff
@@ -155,7 +162,7 @@ void lv_style_reset(lv_style_t * style)
 {
     LV_ASSERT_STYLE(style);
 
-    if(style->is_const) {
+    if(style->prop1 == LV_STYLE_PROP_ANY) {
```

Comment:
`is_const` had to be removed so that the address of `prop1` can be taken. Before, it was a bitfield. Since `LV_STYLE_PROP_ANY` isn't a valid value to store in `prop1`, I used that as a check for a constant style instead.

Reply:
In this case can we add dedicated name for const style? E.g. `_LV_STYLE_PROP_CONST`.

Reply:
I've added it, and under the hood it uses the same value for now.